sschipke/palette-producer
# Ss create menu

## What's this PR do?
- Integrates Redux into the entire application

### Where should the reviewer start?
- All component Files
### How should this be manually tested?
- Try manually **selecting, deleting**, and **creating** palettes and projects. See if the database updates correctly
### What are the relevant tickets?
closes #35 closes #30 closes #28 
#### Screenshots (if appropriate)
